.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,61 @@
+name: 🐞 Bug Report
+description: Report a bug
+labels:
+    - kind/bug
+    - status/needs-triage
+
+body:
+    - type: markdown
+      attributes:
+          value: |
+              ## First of all
+              1. Please search for [existing issues](https://github.com/libgdx/libgdx/issues?q=is%3Aissue) about this problem first.
+              2. Remember to be friendly.
+
+    - type: textarea
+      id: description
+      attributes:
+          label: Describe the bug
+          description: A clear description of what the bug is. Include screenshots if applicable.
+          placeholder: Bug description
+      validations:
+          required: true
+
+    - type: textarea
+      id: reproduction
+      attributes:
+          label: Reproduction
+          description: Steps to reproduce the behavior.
+          placeholder: |
+              1. Go to ...
+              2. Click on ...
+              3. See error
+
+    - type: textarea
+      id: expected-behavior
+      attributes:
+          label: Expected behavior
+          description: A clear description of what you expected to happen.
```

Comment:
I think most people already add that to the description above, so it's not really needed

Reply:
Updated